### ISSUE TITLE
Normalize Input Data

### ISSUE THREAD

**Issue by johnmartinsson** [johnmartinsson/bird-species-classification#15]
Input data should have zero mean and identity variance.

**Note**: mean, and variance should be computed from the training set ONLY, and then subtracted from the training, validation and test sets.

http://cs231n.github.io/neural-networks-2/#datapre

**Comment by johnmartinsson** [comment]
This functionality should be left out for now. But it may be interesting to consider later in the project. Moved to the next milestone.

**Comment by johnmartinsson** [comment]
This could be considered in the future, but will not be part of the thesis work. Removed from milestone.
